wfdb/io/annotation.py

```diff
@@ -1627,10 +1627,7 @@ def field2bytes(field, value, custom_labels=None):
     # samp and sym bytes come together
```

Comment:
Thanks for this. Can you please update the block just above to account for the custom labels?

Reply:
Thanks for your review. Yes, of course. I just committed a fix for this.